Changes

```diff
@@ -1,6 +1,11 @@
 10.4.2.1 (relative to 10.4.2.0)
 ========
 
+Improvements
+------------
+
+- Added string constructor and `fromString` function to `IECore.MurmurHash`.
```

Comment:
This is in the section for an already-released version, but needs to be in a new section. If you want the feature to be released in the next `10.4.x` version, the PR also needs to be targeted to the `RB-10.4` branch.

Reply:
retargeted to RB-10.4